### PR TITLE
Add URLs for storage locations

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

File: core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/StorageLocation.java [hunk 1]
```diff
@@ -117,6 +117,8 @@ public BoxStorageAmount getBoxStorageAmount() {
   @OneToMany(targetEntity = StorageLocationChangeLog.class, mappedBy = "storageLocation", cascade = CascadeType.REMOVE)
   private final Collection<ChangeLog> changeLog = new ArrayList<>();
 
+  private String url;
```
Comment (djcooke) [inline core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/StorageLocation.java]: Please rename `mapUrl` for clarity. Same for DTO and DB column

**Comment by sqwci** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [StorageLocation.java#L283](https://github.com/TGAC/miso-lims/blob/b6fe7f35247eeb8a61955ceb694f730477e51dde/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/StorageLocation.java#L283): Refactor this method to reduce its Cognitive Complexity from 29 to the 15 allowed. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)
1. ![MAJOR][MAJOR] [DefaultStorageLocationService.java#L127](https://github.com/TGAC/miso-lims/blob/b6fe7f35247eeb8a61955ceb694f730477e51dde/miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultStorageLocationService.java#L127): Remove this unused method parameter "beforeChange". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by apmasell** [comment]
Renamed

**Comment by sqwci** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [StorageLocation.java#L283](https://github.com/TGAC/miso-lims/blob/f813c68bfa41b7f2323d1d43a0bb9f3c89738e7e/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/StorageLocation.java#L283): Refactor this method to reduce its Cognitive Complexity from 29 to the 15 allowed. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)
1. ![MAJOR][MAJOR] [DefaultStorageLocationService.java#L127](https://github.com/TGAC/miso-lims/blob/f813c68bfa41b7f2323d1d43a0bb9f3c89738e7e/miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultStorageLocationService.java#L127): Remove this unused method parameter "beforeChange". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**